mineinabyss-components/src/main/kotlin/com/mineinabyss/components/players/Backpack.kt

```diff
@@ -0,0 +1,11 @@
+package com.mineinabyss.components.players
+
+import com.mineinabyss.idofront.serialization.SerializableItemStack
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+@SerialName("mineinabyss:backpack")
+class Backpack(
+    var backpackContent: MutableList<SerializableItemStack>? = mutableListOf()
```

Comment:
Check all components for what I said. This looks like it'd be fine as a mutable list but var makes no sense